charts/seed-terraformer/charts/alicloud-backup/templates/_main.tf

```diff
@@ -21,4 +21,8 @@ resource "alicloud_oss_bucket" "bucket" {
 output "bucketName" {
   value = "${alicloud_oss_bucket.bucket.id}"
 }
+
+output "storageEndPoint" {
```

Comment:
Adding an `output` field won't make `terraform plan` identify a difference.

Can you please add a similar section like https://github.com/gardener/gardener/blob/master/charts/seed-terraformer/charts/openstack-infra/templates/_main.tf#L88-L96 to trigger a `terraform apply` run?

```
// We have introduced new output variables. However, they are not applied for
// existing clusters as Terraform won't detect a diff when we run `terraform plan`.
// Workaround: Providing a null-resource for letting Terraform think that there are
// differences, enabling the Gardener to start an actual `terraform apply` job.
resource "null_resource" "outputs" {
  triggers = {
    recompute = "outputs"
  }
}
```